recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_targets:
+  - conda-forge diffpy.structure_rc
```

Comment:
Instructed in https://conda-forge.org/docs/maintainer/knowledge_base/#creating-a-pre-release-build